docs/API.md

```diff
@@ -451,43 +517,29 @@ Example response:
 
 An object with the following properties:
 
-* `"client_id"`: A free-form string, optional.
-* `"domain"`: A *domain name*, required.
-* `"profile"`: A *profile name*, optional.
-* `"client_version"`: A free-form string, optional.
-* `"nameservers"`: A list of *name server* objects, optional.
-* `"ds_info"`: A list of *DS info* objects, optional.
+* `"client_id"`: A *client id*, optional. Used to monitor which client uses the API.
+* `"domain"`: A *domain name*, required. The zone to test.
+* `"profile"`: A *profile name*, optional. Used to perform the test with a specific set of parameters and tests.
+* `"client_version"`: A *client version*, optional. Used to monitor which client use the API
+* `"nameservers"`: A list of *name server* objects, optional. Used to perform un-delegated test.
+* `"ds_info"`: A list of *DS info* objects, optional. Used to perform un-delegated test.
 * `"advanced"`: **Deprecated**. A boolean, optional.
-* `"ipv6"`: A boolean, optional. (default `false`)
-* `"ipv4"`: A boolean, optional. (default `false`)
-* `"config"`: A string, optional. The name of a *config profile*.
-* `"user_ip"`: A ..., optional.
-* `"user_location_info"`: A ..., optional.
-* `"priority"`: A *priority*, optional
-* `"queue"`: A *queue*, optional
-
->
-> TODO: Clarify the data type of the following `"params"` properties:
-> `"user_ip"` and `"user_location_info"`.
->
-> TODO: Clarify the purpose of each `"params"` property.
->
-> TODO: Clarify the default value of each optional `"params"` property.
->
+* `"ipv6"`: A boolean, optional. (default `false`). Used to configure the test and enable IPv4 tests.
+* `"ipv4"`: A boolean, optional. (default `false`). Used to configure the test and enable IPv6 tests.
+* `"config"`: A string, optional. The name of a config profile.
+* `"user_ip"`: An *IP address*, optional. Used to monitor information about the user. (We only keep the location of the IP).
+* `"user_location_info"`: An *location* object, optional. Used to monitor information about the user. 
+* `"priority"`: A *priority*, optional.
+* `"queue"`: A *queue*, optional.
 
 
 #### `"result"`
 
-A *test id*. The newly started *test*, or a recently run *test* with the same
-parameters.
-started within the recent configurable short time.
-
->
-> TODO: Specify which configuration option controls the duration of the window
-> of *test* reuse.
->
-
+A *test id*. 
 
+If the test has been run with the same domain name within an interval of 10 mins (hard coded), 
+then the new request does not trigger a new test, but returns with the results of the last test
+ 
```

Comment:
Why has it been modified from 5 mins to 10?

Reply:
From the code it looks like it has "always" been 10 minutes. One idea would be to have it configurable, but that should be a new issue.